coord/src/rmr/rq.c

```diff
@@ -26,6 +29,7 @@ typedef struct MRWorkQueue {
   size_t sz;
   uv_mutex_t lock;
   uv_async_t async;
+  volatile atomic_bool isActive;
```

Comment:
Isn't atomic_bool is c++? Do we compile those files with g++? If so which version?

Reply:
c supports bool as of c99 and the atomic library as of c11
https://en.cppreference.com/w/c/thread#Atomic_operations